cypress/e2e/fee-fine/fine-amount-link-in-while-checkout.cy.js

```diff
@@ -0,0 +1,126 @@
+import uuid from 'uuid';
+import moment from 'moment';
+import permissions from '../../support/dictionary/permissions';
+import TopMenu from '../../support/fragments/topMenu';
+import TestTypes from '../../support/dictionary/testTypes';
+import DevTeams from '../../support/dictionary/devTeams';
+import UsersOwners from '../../support/fragments/settings/users/usersOwners';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import Users from '../../support/fragments/users/users';
+import TransferAccounts from '../../support/fragments/settings/users/transferAccounts';
```

Comment:
'TransferAccounts' is defined but never used. Allowed unused vars must match /React/u

Reply:
fixed in the second pr